src/appBootstrapper.jsx

```diff
@@ -17,10 +17,12 @@ import AppConfig from "appConfig";
 import browser from 'bowser';
 import './shared/lib/tracking';
 
-if (localStorage.localdev === 'true') {
+if (localStorage.localdev === 'true' || localStorage.localdist === 'true') {
     __webpack_public_path__ = "//localhost:3000/"
+    localStorage.setItem("e2etest", "true");
```

Comment:
set e2etest to true for devs (when testing google analytics, one can always disable this manually here)